ingeniamotion/wizard_tests/feedbacks_tests/dc_feedback_polarity_test.py

```diff
@@ -0,0 +1,163 @@
+        if self.sensor == SensorType.BISSC2:
+            self.mc.configuration.set_auxiliar_feedback(
+                SensorType.ABS1, servo=self.servo, axis=self.axis
+            )
+            self.logger.info(
+                f"Set velocity and position feedbacks to {self.sensor.name}"
+                f" and axuiliar to {SensorType.ABS1.name}"
+            )
+        else:
+            self.mc.configuration.set_auxiliar_feedback(
+                self.sensor, servo=self.servo, axis=self.axis
+            )
+            self.logger.info(f"Set velocity, position and auxiliar feedbacks to {self.sensor.name}")
+
+    @BaseTest.stoppable
+    def check_movement(self, initial_position: int, current_position: int) -> bool:
+        """Check motor movement
+
+        Args:
+            initial_position: initial position
+            current_position: current position
+
+        Returns:
+            True if positions are different enough, else False
+
+        """
+        return (
+            abs(current_position - initial_position)
+            > self.feedback_resolution * self.MOVEMENT_ERROR_FACTOR
+        )
+
+    @BaseTest.stoppable
+    def increase_current_until_movement(self, initial_position: int, rated_current: float) -> int:
+        """Increase motor current until it moves
+
+        Args:
+            initial_position: initial position
+            rated_current: motor rated current
+
+        Returns:
+            returns final position
+        """
+        for set_curr in self.mc.motion.ramp_generator(
+            0, rated_current, self.CURRENT_RAMP_TOTAL_TIME, self.CURRENT_RAMP_INTERVAL
+        ):
+            self.check_stop()
+            current_pos = self.mc.motion.get_actual_position(servo=self.servo, axis=self.axis)
+            if self.check_movement(initial_position, current_pos):
+                self.mc.motion.motor_disable(self.servo, self.axis)
+                return current_pos
+            self.mc.motion.set_current_quadrature(set_curr, servo=self.servo, axis=self.axis)
+            self.logger.info(f"Set current to {set_curr}")
+        self.mc.motion.motor_disable(self.servo, self.axis)
+        raise TestError(
+            "ERROR: No movement detected. Please, review your feedback configuration & wiring"
+        )
+
+    @staticmethod
+    def calculate_polarity(initial_position: int, final_position: int) -> FeedbackPolarity:
+        """Calculate motor polarity
+
+        Args:
+            initial_position: initial position
+            final_position: final position
+
+        Returns:
+            If final position is smaller than initial position return REVERSED, else NORMAL
+
+        """
+        if final_position - initial_position < 0:
+            return FeedbackPolarity.REVERSED
+        return FeedbackPolarity.NORMAL
+
+    @BaseTest.stoppable
+    def loop(self) -> ResultType:
+        rated_current = self.mc.configuration.get_rated_current(servo=self.servo, axis=self.axis)
```

Comment:
I suggest applying the ramp until de max value between rated current (MOT_RATED_CURRENT) and drive max current (CL_CUR_REF_MAX). Otherwise, if motor current is bigger than drive (i.e in Denali) it would cause a fault